contract/src/main/kotlin/com/r3/corda/sdk/token/contracts/AbstractTokenContract.kt

```diff
@@ -38,43 +36,37 @@ abstract class AbstractTokenContract<T : AbstractToken> : Contract {
         }
     }
 
-    abstract fun handleIssue(
-            group: LedgerTransaction.InOutGroup<T, IssuedTokenType<TokenType>>,
-            issueCommand: CommandWithParties<TokenCommand<*>>
-    )
+    /** Provide custom logic for handling issuance of a token. */
+    abstract fun handleIssue(group: InOutGroup<U, IssuedTokenType<T>>, issueCommand: CommandWithParties<TokenCommand<T>>)
```

Comment:
As mentioned on line 28. I am torn between keeping the signature parallel to the others or expecting a single command.

```suggestion
abstract fun verifyIssue(commands: List<CommandWithParties<TokenCommand<T>>>, inputs: List<U>, outputs: List<U>, tx: LedgerTransaction) {
```